frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -1,18 +1,61 @@
 import * as React from 'react';
+import * as _ from 'lodash';
+import * as classNames from 'classnames';
 import { Modal } from 'patternfly-react';
 import { Button } from '@patternfly/react-core';
 import {
   CatalogItemHeader,
   PropertiesSidePanel,
   PropertyItem,
 } from '@patternfly/react-catalog-view-extension';
+import { CheckCircleIcon } from '@patternfly/react-icons';
 import { Link } from 'react-router-dom';
 import { history, ExternalLink, HintBlock } from '@console/internal/components/utils';
 import { RH_OPERATOR_SUPPORT_POLICY_LINK } from '@console/internal/const';
 import { MarkdownView } from '../clusterserviceversion';
 import { SubscriptionModel } from '../../models';
 import { OperatorHubItem } from './index';
 
+const CapabilityLevel: React.FC<CapabilityLevelProps> = ({ capabilityLevel }) => {
+  const levels = [
+    'Basic Install',
+    'Seamless Upgrades',
+    'Full Lifecycle',
+    'Deep Insights',
+    'Auto Pilot',
+  ];
+  const capabilityLevelIndex = _.indexOf(levels, capabilityLevel);
```

Comment:
nit: no need for lodash here

```suggestion
  const capabilityLevelIndex = levels.indexOf(capabilityLevel);
```